### DISCUSSION TITLE
Intel architecture support?

### DISCUSSION THREAD

**Discussion by Vvitali** [Whisky-App/Whisky#417] [Q&A] [answered]
I tried to update Whisky from 1.1.0 to 1.3.2 through brew, and it told me that x86 is not supported
<img width="1086" alt="image" src="https://github.com/Whisky-App/Whisky/assets/20162801/81cddca4-ac84-4231-97c2-c6bbb8bb83c6">

Are x86-64 CPUs not supported anymore? Only ARM?

I checked the source code of Whisky https://github.com/Whisky-App/Whisky/pull/353 , and it seems related to GPTK Beta 4 not supporting Intel-arch?

**Comment by IsaacMarovitz** [comment, marked as answer]
Since Whisky 1.2.1, x86 CPUs are not supported as they cannot run GPTK beta 4+
